### PR TITLE
POL-983 - Updated sql_service_tier_types instance sizes for GP5

### PR DESCRIPTION
**Pull request by mburmeig**
### Description

Fixes SQL mapping based on Microsoft docs.
Pepsi noticed we were making recommendations to upsize/downsize instances that did not exist in the microsoft docs.
This was causing their recommendations to be incorrect. 
https://azure.microsoft.com/en-us/pricing/details/azure-sql-managed-instance/single/

### Issues Resolved

added correct mapping for GP5

### Link to Example Applied Policy

https://app.flexera.com/orgs/34088/automation/applied-policies/projects/135557?policyId=6573381d55d35d0001a91b4b

### Contribution Check List

- [NA ] New functionality includes testing.
- [NA ] New functionality has been documented in the README if applicable
- [ NA] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: data/azure/sql_service_tier_types.json [hunk 1]
```diff
@@ -123,56 +123,36 @@
       "down": null
     },
     "GP_Gen5_4": {
-      "up": "6",
-      "down": "2"
-    },
-    "GP_Gen5_6": {
       "up": "8",
-      "down": "4"
+      "down": null
```
Comment (XOmniverse) [inline data/azure/sql_service_tier_types.json:127]: Should this be null? Wouldn't this downsize to 2 e.g. GP_Gen5_2?

**Review by XOmniverse — CHANGES_REQUESTED** [review]
Found one possible mistake.

File: data/azure/sql_service_tier_types.json (hunk 1 above)
Comment (mburmeig) [reply data/azure/sql_service_tier_types.json:127]: I was going by the docs noted in the ticket that does not show a 2 instance.
BK noted that we should not use the docs at the URL I posted and we should go by these docs.
https://learn.microsoft.com/en-us/azure/azure-sql/database/resource-limits-vcore-single-databases?view=azuresql

It seems the issue noted is one specific to pepsi and not an issue with the SQL mapping file.

Comment (XOmniverse) [reply data/azure/sql_service_tier_types.json:127]: @mburmeig Then that size shouldn't be listed in the document at all.

Let me know if you've confirmed that this change is unneeded and we can delete this PR.

**Pull request closed without merging**